tests/unit_test/log_test/log_processor_test.py

```
@@ -94,6 +95,263 @@
+    assert query_kwargs["bot"] == "test-bot"
+
+def test_custom_from_to_dates_for_model_testing_logs():
+    from_date = datetime(2025, 8, 1, 0, 0, 0)
+    to_date = datetime(2025, 8, 15, 23, 59, 59)
+    handler = mock_model_testing_handler(kwargs={"from_date": from_date, "to_date": to_date})
+    handler.doc_type.objects = MagicMock()
+
+    try:
+        handler.get_logs_and_count()
+    except Exception:
+        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)